airflow/sensors/base_sensor_operator.py

```diff
@@ -121,7 +122,17 @@ def execute(self, context: Dict) -> None:
                 raise AirflowRescheduleException(reschedule_date)
             else:
                 sleep(self.poke_interval)
-        self.log.info("Success criteria met. Exiting.")
+
+        if isinstance(self, BaseAsyncOperator):
```

Comment:
Oh. Isn't it enough to run the code after the loop in the execute method?

Reply:
I wanted to avoid trying to call `self.process_request` on a `BaseSensorOperator` and getting some no such class method error (as in my current model process_request would only be defined for BaseAsyncOperators).

Reply:
I was doing this to avoid a class method not implemented error.